### DISCUSSION TITLE
NextUI - Select does not work properly when used with Shadcn

### DISCUSSION THREAD

**Discussion by yusuftekinx** [heroui-inc/heroui#2208] [Help]
I want to use shadcn and next-ui together but today I ran into a problem. The problem is that if I open a dialog box with shadcn and put the next-ui select component in it, I cannot make a selection. The selected content area is not clickable. It behaves as if there is a z-index problem. How can I solve this problem?

**Comment by theonlyway** [comment]
I started with shadcn and tried to get both shadcn and nextui working together but basically just ended up moving everything shadcn could do to nextui (I like the idea of shadcn but the base nextui styling is just prettier and saves me a bunch of effort and it pretty extensible). I had a lot of issues with any shadcn components that has a portal primitive as part of the component. For any component that has the portal primitive wrapping another primitive I just made a copy of that component and removed the portal, and it solved any issues I had with selecting things. You could try the same thing.

**Comment by yusuftekinx** [comment]
Thanks for the reply

**Comment by monterourena** [comment]
Here is another workaround that may be useful if you have this problem: https://github.com/nextui-org/nextui/discussions/3553#discussioncomment-10144077
